tailwind.config.ts

```diff
@@ -12,6 +12,13 @@ const config = withUt({
   ],
   prefix: "",
   theme: {
+    container: {
+      center: true,
+      padding: "2rem",
+      screens: {
+        "2xl": "1400px",
+      },
+    },
```

Comment:
I think it should be inside extend below, otherwise the default stuff will be removed + why we need it anyway?????